src/main/kotlin/no/nav/familie/ef/sak/fagsak/SøkService.kt

```diff
@@ -127,13 +135,17 @@ class SøkService(
             )
         }
 
-        val personSøkResultat = pdlSaksbehandlerClient.søkPersonerMedSammeAdresse(søkeKriterier)
+        val personSøkResultat = pdlSaksbehandlerClient.søkPersonerMedSammeAdresse(søkeKriterier).hits
 
         return SøkeresultatPerson(
-            hits = personSøkResultat.hits.map { tilPersonFraSøk(it.person) },
-            totalHits = personSøkResultat.totalHits,
-            pageNumber = personSøkResultat.pageNumber,
-            totalPages = personSøkResultat.totalPages,
+            personer =
+                personSøkResultat
+                    .map { tilPersonFraSøk(it.person, grunnlag) }
+                    .sortedWith(
+                        compareByDescending<PersonFraSøk> { it.erSøker }
+                            .thenByDescending { it.erBarn }
```

Comment:
hva kommer først av true og false ved synkende sortering av boolean? Og kan it.erBarn være null?

Reply:
True sjekkes først, deretter mot false og null - trenger derfor ikke å håndtere erBarn og erSøker null.